frontend/app/templates/components/mapbox/current-mouseover-features.hbs

```diff
@@ -0,0 +1,22 @@
+<Mapbox::CurrentMousePosition
+  @map={{this.map}}
+  @handleMousePosition={{this.setPoint}}
+as |mousePoint|>
+  <Mapbox::IntersectingFeatures
+    @map={{this.map}}
+    @point={{mousePoint}}
+    @options={{hash layers=(array this.layerId)}}
+    @handleIntersectingFeatures={{this.setFeatures}}
+  as |intersectingFeatures|
+  >
+    {{#if (and mousePoint (and intersectingFeatures this.mouseoverFeatures))}}
+      {{yield (hash 
+          features=this.mouseoverFeatures
+          point=mousePoint
+        )
+      }}
+    {{else}}
+      {{yield null}}
```

Comment:
is this necessary? if the condition isn't met, it just won't yield the block. if it still needs to yield a block, it could be just be {{yield}}. Not sure I follow...

Reply:
Oops, yeah unnecessary. Originally I was thinking it would be good to have a predictable yield value if the condition evaluated to true, but it looks like if there's no yield then the `as |arg|` is null anyway.